### PR TITLE
Add Blender documents

### PR DESCRIPTION
**Pull request by hologerry**
Add Blender manual and Python API documents.

### CODE REVIEW

File: docsets/bpy/docset.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "archive": "Blender.tgz",
+    "author": {
+        "name": "Yue Gao",
+        "link": "https://github.com/hologerry"
+    },
+    "aliases": ["Blender", "Blender Python API"],
+    "specific_versions": [
+        {
+            "version": "3.6",
+            "archive": "versions/3.6/Blender.tgz"
```
Comment (Kapeli) [inline docsets/bpy/docset.json:13]: This should be `versions/3.6/bpy.tgz`?

Comment (hologerry) [reply docsets/bpy/docset.json:13]: Yeah, I will update this.

**Comment by hologerry** [comment]
I have updated the json and icon size.

**Comment by Kapeli** [comment]
Thank you!

**Pull request merged**